src/volttron/utils/__init__.py

```diff
@@ -68,10 +65,47 @@ def load_config(config_path):
     try:
         with open(config_path) as f:
             return yaml.safe_load(f.read())
-    except yaml.scanner.ScannerError as e:
+    except yaml.YAMLError as e:
         try:
             with open(config_path) as f:
                 return parse_json_config(f.read())
         except Exception as e:
             _log.error("Problem parsing agent configuration")
             raise
+
+
+def update_kwargs_with_config(kwargs, config):
+    """
+    Loads the user defined configurations into kwargs.
+
+      1. Converts any dash/hyphen in config variables into underscores
+      2. Checks for configured "identity" value. Prints a deprecation
+      warning and uses it.
+      3. Checks for configured "agentid" value. Prints a deprecation warning
+      and ignores it
+
+    :param kwargs: kwargs to be updated
+    :param config: dictionary of user/agent configuration
+    """
+
+    if config.get("identity") is not None:
```

Comment:
since we are at 10.0 we can probably remove the warnings. and just not pick the identities up...same with agentid

Reply:
done